pkg/features/kube_features.go

```diff
@@ -943,7 +944,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	JobMutableNodeSchedulingDirectives:             {Default: true, PreRelease: featuregate.Beta},
 	IdentifyPodOS:                                  {Default: false, PreRelease: featuregate.Alpha},
 	PodAndContainerStatsFromCRI:                    {Default: false, PreRelease: featuregate.Alpha},
-	HonorPVReclaimPolicy:                           {Default: false, PreRelease: featuregate.Alpha},
+	HonorPVReclaimPolicy:                           {Default: false, PreRelease: featuregate.Beta},
```

Comment:
@jsafrane This is not enabled by default?  I thought only an API object is subject to this new change that Beta means disabled by default.

Reply:
This is from the email announcing Beta API off by default: https://groups.google.com/a/kubernetes.io/g/dev/c/tkzbBcS0JI8
`Beta feature gates remain on by default and beta fields on stable APIs remain on by default. `

Reply:
you're right, it should be enabled by default. @deepakkinni, can you please fix it?

Reply:
Sure, I'll work on it.